### DISCUSSION TITLE
Is there is size limit when uploading mp4 or mov files in Github issues/pull requests?

### DISCUSSION THREAD

**Discussion by hhimanshu** [community/community#4772] [Projects and Issues] [answered]
Hello there,

I am wondering if there is size limit for an individual video file that can be attached to a Github issue or a pull request

Please let me know
Thank you

**Comment by danpoynor** [comment, marked as answer]
The limit might be the same as for adding video to a readme - 10 MB, or 100 MB if you have a Pro account.

**Reply by rcowsill** [reply]
That's right according to the documentation:
https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files

It seems to depend on the free/pro status of the repo you're posting the video to, rather than your own account.
